libcni/api.go

```diff
@@ -193,24 +206,78 @@ func (c *CNIConfig) getResultCacheFilePath(netName string, rt *RuntimeConf) stri
 	return filepath.Join(c.getCacheDir(rt), "results", fmt.Sprintf("%s-%s-%s", netName, rt.ContainerID, rt.IfName))
 }
 
-func (c *CNIConfig) setCachedResult(result types.Result, netName string, rt *RuntimeConf) error {
+func (c *CNIConfig) setCachedResult(result types.Result, config []byte, netName string, rt *RuntimeConf) error {
+
+	cached := cachedInfo{
+		Config:     config,
+		Kind:       "cniCacheV1",
+		CNIVersion: "0.4.0"}
```

Comment:
I'm not sure about this one, but probably it should also be declared as constant. Is it anywhere used?